tegel-styles/scss/ag-grid/tegel-ag-grid-styles.scss

```diff
@@ -0,0 +1,166 @@
+/* AG-Grid General Overrides using Tegel Design System Variables */
```

Comment:
I'm thinking the "ag-grid"-prefix is quite redundant here? Maybe even the "ag-grid" one? Could also just be:
`/ag-grid/styles.scss`?
Or `overrides.scss`?

Reply:
Update done!